Examples/Backend/DataAdapters/DataAdapter-SharePoint/Assets/Common/SharepointContentConverter.cs

```diff
@@ -333,6 +338,10 @@ private void SetThumbnails(AssetDataObject assetDataObject, DriveItem driveItem)
             {
                 var thumbnailSet = driveItem.Thumbnails?.FirstOrDefault();
 
+                assetDataObject.InternalMetadata.SmallThumbnailETag = GetETag(thumbnailSet?.Small?.Url);
```

Comment:
When `IsPlaybackLargeAvailable == true` unfortunately we are using the Graph API and we don't have the URL
The same is true when `IsPdfPreviewAvailable == true` and `IsThumbnailPreviewAvailable == true`